client/verta/verta/_cli/registry/update.py

```diff
@@ -39,15 +39,26 @@ def update_model(model_name, label, workspace, description):
 @click.argument("version_name", nargs=1, required=True)
 @click.option("--label", "-l", multiple=True, help="Label to be associated with the object.")
 @click.option("--model", help="Path to the model.")
+@click.option("--custom-module", type=click.Path(exists=True), multiple=True, help="Path to custom module file or directory.")
+@click.option("--no-custom-modules", help="Flag to not upload any custom modules.", is_flag=True)
 @click.option("--artifact", type=str, multiple=True, help="Path to the artifact required for the model. The format is --artifact artifact_key=path_to_artifact.")
 @click.option("--workspace", "-w", help="Workspace to use.")
 @click.option('--overwrite', help="Overwrite model and artifacts if already logged.", is_flag=True)
 @click.option("--requirements", type=click.Path(exists=True, dir_okay=False), help="Path to the requirements.txt file.")
 @click.option("--description", "-d", help="Description.")
-def update_model_version(model_name, version_name, label, model, artifact, workspace, overwrite, requirements,
+def update_model_version(model_name, version_name, label, model, custom_module, no_custom_modules, artifact, workspace, overwrite, requirements,
                          description):
     """Update an existing registeredmodelversion entry.
     """
+    if custom_module and no_custom_modules:
+        raise click.BadParameter("--custom-module cannot be used alongside --no-custom-modules.")
+    elif no_custom_modules:
+        custom_modules = []
+    elif custom_module:
+        custom_modules = list(custom_module)  # listify tuple. Is this necessary?
```

Comment:
I haven't verified this, but I'd expect a `tuple` to be fine. Did you encounter any particular issues when not casting it to a `list`?

Reply:
Nope no issue at all. Fixed.